proposals/0002-cxx-attributes.md

```diff
@@ -228,7 +228,125 @@ following grammar formulations are valid:
   \terminal{using} \textit{identifier} \opt{attribute-specifier-seq} \terminal{=} \textit{type-id} \terminal{;}\br
 \end{grammar}
 ```
-
 ![Latex Rendering](0002-assets/ClassGrammarRender.png)
 
+### Attribute Specification Language
+
+Attributes annotate source constructs with information. An attribute is said to
+be _applied to_ the entity or statement identified by the source construct.
+
+Some attributes may be required by an implementation for correct code
+generation, others may be optionally ignored. An implementation must issue a
+diagnostic on all ignored attributes unless otherwise specified by the
+definition of the attribute behavior.
+
+> Note: an example here would be optimization hint attributes which an
+> implementation is allowed to ignore without diagnosing.
+
+Each attribute may specify specific behavior for parsing attribute arguments.
+Any attribute that does not specify specific behavior shall be parsed with the
+general behavior described here.
+
+An empty attribute specifier has no effect. The order in which attributes
+applied to the same source construct are written shall not be significant. When
```

Comment:
So this means attribute overriding/duplication/conflict will be determined by each attribute definition?

Like if I did `uint field [[hlsl::system_value(something), hlsl::system_value(something_else)]]`